tdrs-backend/tdpservice/users/admin.py

```diff
@@ -35,9 +35,11 @@ class UserAdmin(admin.ModelAdmin):
     exclude = ['password', 'user_permissions', 'is_active']
     readonly_fields = ['last_login', 'date_joined', 'login_gov_uuid', 'hhs_id', 'access_request', 'deactivated']
     form = UserForm
-    list_filter = ('account_approval_status', 'stt', 'region')
+    list_filter = ('account_approval_status', 'region', 'stt')
     list_display = [
         "username",
+        'location',
```

Comment:
i believe `location` here is redundant now that both `stt` and `region` are displayed

Reply:
Yep merge conflict issue